resources/config.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Worlds where entities will not be allowed to spawn automatically.
+# eg Disabled-Worlds: [world, lobby, arena, home]
+Disabled-Worlds: []
+
+# Default setting for all-mobs when creating protected areas in centers.yml.
+# True means that all mobs will be blocked from spawning inside the protected area. 
+All-Mobs: true;
```

Comment:
random semicolon?

Reply:
Fixed in https://github.com/95CivicSi/PEXProtection/commit/b36bf392a60e7441e27b39e1ffd3b233f277db74